sabelotodo/routes.py

```diff
@@ -1,6 +1,5 @@
-from flask import current_app as app
+from flask import abort, jsonify, current_app as app
```

Comment:
hmmm.... I wonder what Flask's jsonify method does and how it's different from the native `json.dumps` (oh, it probably handles classes at all.) I know there's a security issue with the `jsonify` package, but this may not use it. :thinking: